src/DummyTest.java

```diff
@@ -143,6 +143,74 @@ public void exceptionTestingHealth() {
         // Assert
         assertThrows(IllegalArgumentException.class, executable, "Exception handling failed");
     }
+
+    @Test
+    @DisplayName("Test for battle function")
+    void testForBattle1() {
+        // Arrange
+        Entity entity1 = new Entity(100, 50, 35, 2, "Bob", "Bob story");
+        Entity entity2 = new Entity(90, 40, 30, 2, "Mike", "Mike story");
+        var printer = System.out;
+        Battle battle = new Battle(printer);
+
+        // //Act
+        battle.battle(entity1, entity2);
```

Comment:
A `Battle#battle` metódusnak van egy visszatérési értéke, ami a harc kimenetelét jelképezi. Azt is nézni kell. Nem konkrétan itt, de legyen arra is teszt.